lib/rules/sotd/pp.js

```diff
@@ -110,6 +130,15 @@ exports.check = function (sr, done) {
             return done();
         }
 
+        // Use Promise.all to get Group Names, if groups include IG, will check `noRec` sentence in patent policy paragraph.
+        await Promise.all(fetchGroupNames(sr.getDelivererIDs())).then(function (groupNames) {
+            sr.isIGDeliverable = groupNames.some(groupName => {
+                return groupName.endsWith('Interest Group');
+            });
```

Comment:
```suggestion
            sr.isIGDeliverable = groupNames.some(groupName => groupName.endsWith('Interest Group'));
```

Reply:
Thanks for the reminder. Update for `doc/wg-note2.html` is adding two delivererID, but it shouldn't be in this paragraph. Will move to another paragraph.